### PR TITLE
[TACHYON-1602] Remove TachyonException#getType

### PR DESCRIPTION
**Pull request by sisirkoppaka**
https://tachyon.atlassian.net/projects/TACHYON/issues/TACHYON-1602

### CODE REVIEW

File: clients/unshaded/src/main/java/tachyon/client/worker/BlockWorkerClient.java [hunk 1]
```diff
@@ -286,7 +287,7 @@ public LockBlockResult call() throws TachyonTException, TException {
         }
       });
     } catch (TachyonException e) {
-      if (e.getType() == TachyonExceptionType.FILE_DOES_NOT_EXIST) {
+      if (e instanceof FileAlreadyExistsException) {
```
Comment (aaudiber) [inline clients/unshaded/src/main/java/tachyon/client/worker/BlockWorkerClient.java]: should this be `FileDoesNotExistException`?

**Comment by aaudiber** [comment]
Thanks for this improvement! Do you mind adding a link to the Jira ticket in the PR description? I've left a comment on the code.

**Comment by sisirkoppaka** [comment]
@aaudiber Thanks! I added the link to the Jira ticket above. 
I was afraid of exactly this kind of a mistake :) Thanks a lot for catching it. I pushed the change

**Comment by aaudiber** [comment]
LGTM, thanks for fixing

**Pull request merged**